eng/Versions.props

```diff
@@ -221,7 +221,7 @@
     <!-- ICU -->
     <MicrosoftNETCoreRuntimeICUTransportVersion>8.0.0-rtm.23523.2</MicrosoftNETCoreRuntimeICUTransportVersion>
     <!-- MsQuic -->
-    <MicrosoftNativeQuicMsQuicSchannelVersion>2.3.5</MicrosoftNativeQuicMsQuicSchannelVersion>
+    <MicrosoftNativeQuicMsQuicSchannelVersion>2.3.8</MicrosoftNativeQuicMsQuicSchannelVersion>
```

Comment:
Should this be 2.4.8? or are we keeping 2.3.* on purpose?

Reply:
Fixed, good catch.